zio-http/src/test/scala/zhttp/service/ServerSpec.scala

```diff
@@ -169,9 +174,13 @@ object ServerSpec extends HttpRunnableSpec(8088) {
   private val env = EventLoopGroup.nio() ++ ChannelFactory.nio ++ ServerChannelFactory.nio ++ AppCollection.live
 
   private val staticApp = Http.collectM[Request] {
-    case Method.GET -> !! / "success"       => ZIO.succeed(Response.ok)
-    case Method.GET -> !! / "failure"       => ZIO.fail(new RuntimeException("FAILURE"))
-    case Method.GET -> !! / "get%2Fsuccess" => ZIO.succeed(Response.ok)
+    case Method.GET -> !! / "success"                => ZIO.succeed(Response.ok)
+    case Method.GET -> !! / "failure"                => ZIO.fail(new RuntimeException("FAILURE"))
+    case Method.GET -> !! / "get%2Fsuccess"          => ZIO.succeed(Response.ok)
+    case req @ Method.POST -> !! / "echoRequestBody" =>
+      for {
+        chunk <- req.getBody
+      } yield Response.text(chunk.map(_.toChar).mkString(""))
```

Comment:
Avoid writing tests on `staticApp`. They are hard to maintain and not co-located

Reply:
Done, moved to DynamicAppSpec, but under separate suite for testing getBody, not to mess with the existing tests

Reply:
Later in a separate PR when I will add support for setBody, I will try to organise tests under same suite for all Request APIs